### PR TITLE
feat: slack notifier added

### PR DESCRIPTION
**Pull request by AswinTimalsina**
If merged, please change `push/branches/slack-test` to `push/branches/main`. I have used `slack-test` branch for testing purpose.

For this to work, integration should be added to the channel. `SLACK_WEBHOOK` can be generated from https://slack.com/apps/A0F7XDUAZ-incoming-webhooks 

Please let me know if anything needs to be changed.

Thanks!

Signed-off-by: Aswin Timalsina <aswin.timalsina1@gmail.com>

### CODE REVIEW

**Comment by AswinTimalsina** [comment]
It looks like this:
image

I could change a lot of things. 
image

The image is from https://github.com/marketplace/actions/slack-notify

**Comment by jkutner** [comment]
@AswinTimalsina  For now, let's set `SLACK_MESSAGE` to:

> A namespace has been updated or created

In future work, we can improve it to be more specific, and even include the name of the namespace.

File: .github/workflows/namespace-notifier.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    name: Slack Channel Notify
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Slack Channel Notify
+      uses: rtCamp/action-slack-notify@v2.1.2
+      env:
+        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+        SLACK_USERNAME: ${{ github.repository }}
+	SLACK_MESSAGE: A namespace has been updated or created.
```
Comment (jkutner) [inline .github/workflows/namespace-notifier.yml:18]:
```suggestion
        SLACK_MESSAGE: A namespace has been updated or created.
```

yaml....

File: .github/workflows/namespace-notifier.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: Buildpack Registry Namespace Notifier
+on: 
+  push:
+    branches:
+      - slack-test
```
Comment (jkutner) [inline .github/workflows/namespace-notifier.yml:5]:
```suggestion
      - main
```

Let's go ahead and set this to main when we merge it. The action will fail at first, but that's fine.

**Review by jkutner — APPROVED** [review]

**Pull request merged**